src/app/core/entity-components/entity-form/entity-form/entity-form.component.ts

```diff
@@ -97,7 +98,9 @@ export class EntityFormComponent<T extends Entity = Entity> implements OnInit {
       this.save.emit(this.entity);
       this.switchEdit();
     } catch (err) {
-      this.alertService.addWarning(err.message);
+      if (!(err instanceof InvalidFormFieldError)) {
```

Comment:
Should we make this consistent then? The same code is used in the `RowDetails` and the `EntitySubrecord` component.

Reply:
thanks, good suggestion! (And we should really try to merge some of that code)

implemented this now